R/read_absmaps.R

```diff
@@ -47,6 +50,10 @@ read_absmap <- function(name = NULL,
     warning("Both name and area/year entered. Defaulting to name value: ", name)
   }
 
```

Comment:
(Genuine Q, not sure the answer) Do you want to add some kind of checking to see that the name (or area/year combination) is a valid one?

Reply:
I thought about that, but wanted to have a single point of truth (so that if there were items added to the `absmapsdata` package, they could be accessed through `strayr::read_absmap()` without having to update the `strayr` package. And I didn't want to add an internet-dependent check in the case where the file is found locally.

But it's possible to meet those conditions -- I'll add it to this PR

Reply:
Yeah fair enough